### PR TITLE
Update changelog.md and add use case for extending manager clusterrole

### PR DESCRIPTION
**Pull request by OsamaKhan220**

### CODE REVIEW

**Comment by stakater-user** [comment]
@OsamaKhan220 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-272-11d1c16e`

**Comment by stakater-user** [comment]
@OsamaKhan220 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-272-bf740ab2`

File: content/sre/tenant-operator/usecases/manager-clusterrole.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Extending Manager ClusterRole
+
+Bill as the cluster admin want to add addtional rules for manager CulsterRole.
```
Comment (MuneebAijaz) [inline content/sre/tenant-operator/usecases/manager-clusterrole.md]: `ClusterRole`^

**Comment by stakater-user** [comment]
@OsamaKhan220 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-272-9f6b6873`

**Review by MuneebAijaz — APPROVED** [review]
LGTM

**Comment by stakater-user** [comment]
@MuneebAijaz Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.210`

**Pull request merged**